base/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM fedora:27
+#FROM fedora:27
```

Comment:
Do you still need it or we can just remove it?

Reply:
I just kept it to indicate that we are in general using fedora version 27.